core/src/main/java/org/apache/calcite/sql/validate/SqlConformance.java

```diff
@@ -303,6 +303,17 @@ public interface SqlConformance {
    */
   boolean isInsertSubsetColumnsAllowed();
 
+  /**
+   * Whether the dialect is able to ORDER BY literal values.
```

Comment:
2nd ('This flag') and 3rd ('Returns false') paragraphs need to start with `<p>`.